src/common-utils/main.sh

```diff
@@ -426,7 +426,24 @@ if [ "${RC_SNIPPET_ALREADY_ADDED}" != "true" ]; then
     RC_SNIPPET_ALREADY_ADDED="true"
```

Comment:
Can we bump the patch version? Thanks! 
https://github.com/devcontainers/features/blob/6e635a6ea0afee640bf51adcb5a758e39c7dcbea/src/common-utils/devcontainer-feature.json#L3